project_generation/content/templates/event/config/config.go.tmpl

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config represents service configuration for {{.Name}}
+type Config struct {
+	BindAddr                   string        `envconfig:"BIND_ADDR"`
+	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
+	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	KafkaAddr                  []string      `envconfig:"KAFKA_ADDR"                     json:"-"`
+	KafkaVersion               string        `envconfig:"KAFKA_VERSION"`
+	KafkaOffsetOldest          bool          `envconfig:"KAFKA_OFFSET_OLDEST"`
+	HelloCalledGroup           string        `envconfig:"HELLO_CALLED_GROUP"`
+	HelloCalledTopic           string        `envconfig:"HELLO_CALLED_TOPIC"`
+}
+
+var cfg *Config
+
+// Get returns the default config with any modifications through environment
+// variables
+func Get() (*Config, error) {
+	if cfg != nil {
+		return cfg, nil
+	}
+
+	cfg = &Config{
+		BindAddr:                   "localhost:{{.Port}}",
+		GracefulShutdownTimeout:    5 * time.Second,
+		HealthCheckInterval:        30 * time.Second,
+		HealthCheckCriticalTimeout: 90 * time.Second,
+		KafkaAddr:                  []string{"localhost:9092"},
+		KafkaVersion:               "1.0.2",
+		KafkaOffsetOldest:          true,
+		HelloCalledGroup:           "{{.Name}}",
+		HelloCalledTopic:           "hello-called",
```

Comment:
No, this is not a project specific value. All example event handler projects have a "hello-called" topic initially as the example. This is similar to the way all API projects have a HelloWorld handler or all controllers have a HelloWorldController. You could argue that the HelloCalledGroup value doesn't need to be parameterised either as it will be replaced by an app specific value anyway, but it does show that conventionally this is named after the app name as this will have been filled in at project creation time.